### DISCUSSION TITLE
dropdown menu adding atribute to the body and moving the background image of a section

### DISCUSSION THREAD

**Discussion by wellington-vell** [shadcn-ui/ui#4637] [Q&A]
i copy and paste the mode toggle from the doc and when i am opening the dropdown this happens and data-scroll-locked="1" is added to the body

https://github.com/WellingtonVell/portfolio

**Comment by MithrilMan** [comment]
In my case I fixed by defining `modal={false}` on the DrowpownMenu
`<DropdownMenu modal={false}>`

**Reply by ZakariAdamu** [reply]
Awesome. This solution works for me too
